parachains/runtimes/bridge-hubs/test-utils/Cargo.toml

```diff
@@ -0,0 +1,50 @@
+[package]
+name = "bridge-hub-test-utils"
+version = "0.1.0"
+authors = ["Parity Technologies <admin@parity.io>"]
+edition = "2021"
+description = "Utils for BridgeHub testing"
+
+[dependencies]
```

Comment:
Nit: Do we need all the other dependencies apart from `asset-test-utils` ?

Reply:
ok, removed, they will come later [here](https://github.com/paritytech/cumulus/pull/2111/files#diff-b70b5564f1d2014dc102ea781326a768361b8ccb9faacfc3517fc1acb3e99814)